src/Components/AddBook/AddBook.tsx

```diff
@@ -52,6 +53,8 @@ const AddBook = () => {
     console.log(data)
   }, [data])
 
+  if (error) return <ServerError />
```

Comment:
Nice logic! In JS for better readability, we generally want to wrap this sort of logic either in a function or in our return statement instead of having it floating. The best practice is to include it in a return statement like this: 
![image](https://user-images.githubusercontent.com/108096652/218493719-72983247-109e-440e-bed0-f4027ceee15c.png)
I think it might be worth going through and refactoring before we merge!

Reply:
Here is a good resource that outlines what the `&&` is doing under the hood: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND